packages/wizard/src/lib/Tab/index.tsx

```diff
@@ -0,0 +1,52 @@
+import CheckIcon from '@mui/icons-material/Check';
+import Typography from '@mui/material/Typography';
+
+import { Container, TextContainer } from './styles';
+import { TabProps } from '../types';
+
+const Tab = (props: TabProps) => {
+  const { step, isCompleted, showCompleted, TabOverride, isActive } = props;
+
+  const { components, title, subtitle, disabled, ...others } = step;
+
+  const tabOverrideProps = { ...props };
+  delete tabOverrideProps.TabOverride;
+
+  const disabledTextColor = disabled ? 'gray' : '';
+
+  if (components?.Tab) {
+    return components.Tab({
+      step: { title, subtitle, disabled, isActive, ...others },
+    });
+  }
+
+  if (TabOverride) {
+    return <TabOverride {...tabOverrideProps} />;
+  }
+
+  return (
+    <Container disabled={disabled}>
+      <TextContainer>
+        <Typography variant="body1" color={disabledTextColor}>
+          {title && title}
```

Comment:
try to use ternary instead of && 

a good explanation why [here](https://kentcdodds.com/blog/use-ternaries-rather-than-and-and-in-jsx)